content/_includes/site/sponsor-us.njk

```diff
@@ -0,0 +1,23 @@
+{% import 'embed.macros.njk' as embed %}
+
+<aside data-layout="cta" aria-labelledby="hire-cta-title" class="hire-us-cta" data-ccs-colors="invert">
+  {% call embed.media_block(
+    name='sponsor-cta',
+    media=embed.screen(
+      fill='#dbd8df',
+      content=embed.img(
+        src='work/adobe/luxury-mobile.jpg',
+        alt='Mobile view of the luxury font tag question',
+        sizes='media'
+      )
+    )
+  ) %}
+    <h2 id="hire-cta-title" data-header='stacked hire-cta'>
+      <span class="title-action">Hire Us to Develop</span>
+      Languages & Tools
+      <span class="title-description">That you depend on</span>
+    </h2>
+
+    <a href='/contact/' data-btn="link-button" role="button" class="section-link" draggable="false">Let's chat</a>
```

Comment:
@mirisuzanne we added the `data-ccs-colors="invert"` to the parent container (aside) but this also inverts the button, which we don't want. If we add `data-ccs-colors` without any value to this button, does that work? It seemed to in my local testing but I wanted to check with you cc: @SondraE

Reply:
From the designs, it looks to me like this section is always light-on-dark? I would expect `data-ccs-colors="dark"` to be the best choice here. And ideally the link-button styles should adapt based on the theme. "Invert" will sometimes be dark-on-light, because it uses the reverse of whatever the user has set for the page.

Aside: why a link with role button, and draggable set to false?

Reply:
> From the designs, it looks to me like this section is always light-on-dark?

@mirisuzanne I think she changed her mind yesterday and wanted to use the site's heading gradient for the background of this callout container, but she wanted to invert it so the background doesn't stand out as much. This also means it should change based on light or dark mode. Maybe we just need to assign new colors for this background instead of trying to use the same foreground color in the opposite color mode. 

No idea on the button question. I think this is copied as-is from the hire-us.njk file

Reply:
> Aside: why a link with role button, and draggable set to false?

@mirisuzanne That was my doing. Since it's a link styled as a button, I followed [this](https://www.tempertemper.net/blog/when-design-breaks-semantics) and tried to make it behave as much as a button as possible.

Reply:
Hmmm, I'm skeptical for the reasons they mention at the end. If there weren't alternatives, I'd understand. But it seems to me like a better approach would be to create a "high-priority link" style that is still clearly a link, but maybe on a background, instead of looking like a button at all.